libraries/k8s_backend.rb

```diff
@@ -42,3 +42,66 @@ def catch_k8s_errors
     nil
   end
 end
+
+module Validators
+  # @see https://github.com/inspec/inspec-aws/blob/master/libraries/aws_backend.rb#L209
+  #
+  # @param opts [Hash] The parameters to be validated.
+  # @param resource_name [String] The name of the method/resource that the parameters are validated in.
+  # @param allow [Array] The list of optional parameters.
+  # @param required [Array] The list of required parameters.
+  # @param require_any_of [Array] The list of parameters that at least one of them are required.
+  def self.validate_parameters(resource_name: nil, allow: [], required: nil, require_any_of: nil, opts: {}, skip_length: false) # rubocop:disable Metrics/ParameterLists
```

Comment:
As required and required_any_of is an array should that be default to [] instead of nil?